### DISCUSSION TITLE
Many questions for configuration

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#1986] [Help with using Postal]
Hello everyone I have 3 questions I hope you can help me with this 🙏

1) I want to use port 2525 on Postal, when I make the modification on "postal.yml" it works, but thereafter on Postal all that is return messages (for example: reply or answer) does not pass any more because it absolutely wants to use the port 25, would you have a solution so that Postal uses the port 2525 in all the cases?

2) About SPF records how to do when using ip pools because having followed tutorials on the internet about this I still find myself when I send with SPF: none and the spf.postal.*****.fr is not recognized during the tests (mail-tester and other sites advise me to add an SPF record to Psrp but it changes nothing)

3) I want to add workers to Postal when modifying /opt/postal/config/postal.yml I add "workers: quantity: 3 threads: 4" however when I restart postal the number of workers does not increase with postal status I only see postal_worker_1 how could I do to have my 3 workers? Without using the command "postal start --scale worker=3"

**Comment by willpower232** [comment]
1. Even if you configure Postal to use port 2525 it has to receive on port 25 because that is the standard so bounces and incoming messages will not arrive. You can configure something like iptables to route traffic on port 2525 to port 25 to effectively open both but why do you need 2525?

2. You should add the IP pool addresses to whatever SPF record you're using

3. You cannot scale workers any more using the config file as the processes are controlled by Docker. If you don't want to manually use the scale command, you could duplicate the worker entry in the docker-compose.yml file but that wouldn't be as easy as the scale command

**Reply by Orminor77** [reply]
1) I would like to use port 2525 because port 25 in France is frowned upon by ISPs because it is often used by spammers for scamming and malicious purposes, would you have a tutorial on hand to configure the IP table please?

2) Thanks for the answer, also do you have a sample SPF record to comply with Postal standards?

3) Very well, I will stay with the first method, it works very well for me, thanks for the answer

**Comment by Orminor77** [comment]
Hello, i find the solution for all now, IP table work well thanks you !

I have a last question please, I wanna use only IPV6 to send emails and after looking many posts no answer help me, so how can I use IPV6 in pools IP and ignore IPV4? Please

**Reply by willpower232** [reply]
I don't think that is currently possible I'm afraid

**Reply by Orminor77** [reply]
Ok thanks anyway for your help 👍
